README.md

```diff
@@ -39,3 +43,36 @@ Resource group exists:
         organization: EITR Technologies
     - connection_auth: {{ profile }}
 ```
+
+# Security
+
+To make the file more secure, use the acct comand to encrpt the file with the Fernet algorithm
+
+~~~
+(env) $ acct myawesomecreds.yml
+New encrypted file at:myawesomecreds.yml.fernet
+The file was encrypted with this key:
+71Gbz2oDSv40Er9YUFBJPzOjtCi6Z2-5niBHPekkvqs=
+~~~
+Now we have an encryted file containing the file and a symmetric key. You can use that key to decrpyt the encrypted file.
+
+Since you have the encypted file with the key you can remove the original file
+
+~~~
+(env) $ rm my awesomecreditals.yml
+
+~~~
+
+ # Setting up Envirmoent Variables
+
+ All we ahve to do now is to tell idem where to get the file and key for acct. So we will set up envirmoemtn variables to do those tasks
+
+~~~
+(env) $ export ACCT_FILE="<location_of_ferment_file>"
+(env) $ export ACCT_KEY="1Gbz2oDSv40Er9YUFBJPzOjtCi6Z2-5niBHPekkvqs="
+~~~
+
+# Testing
+~~~
+(env) $ idem state mytest.sls
```

Comment:
this command will actually build the infrastructure. maybe change the name to "Testing and Building" and then show an example with `--test` before actually building